modules/rules_rust/0.35.0/MODULE.bazel

```diff
@@ -0,0 +1,141 @@
+    "cui__pathdiff-0.2.1",
+    "cui__regex-1.10.2",
+    "cui__semver-1.0.20",
+    "cui__serde-1.0.190",
+    "cui__serde_json-1.0.108",
+    "cui__serde_starlark-0.1.14",
+    "cui__sha2-0.10.8",
+    "cui__spectral-0.6.0",
+    "cui__tempfile-3.8.1",
+    "cui__tera-1.19.1",
+    "cui__textwrap-0.16.0",
+    "cui__toml-0.8.6",
+    "cui__tracing-0.1.40",
+    "cui__tracing-subscriber-0.3.17",
+    "generated_inputs_in_external_repo",
+    "libc",
+    "llvm-raw",
+    "rrra__anyhow-1.0.71",
+    "rrra__clap-4.3.11",
+    "rrra__env_logger-0.10.0",
+    "rrra__itertools-0.11.0",
+    "rrra__log-0.4.19",
+    "rrra__serde-1.0.171",
+    "rrra__serde_json-1.0.102",
+    "rules_rust_bindgen__bindgen-0.65.1",
+    "rules_rust_bindgen__bindgen-cli-0.65.1",
+    "rules_rust_bindgen__clang-sys-1.6.1",
+    "rules_rust_bindgen__clap-4.3.3",
+    "rules_rust_bindgen__clap_complete-4.3.1",
+    "rules_rust_bindgen__env_logger-0.10.0",
+    "rules_rust_prost",
+    "rules_rust_prost__h2-0.3.19",
+    "rules_rust_prost__heck",
+    "rules_rust_prost__prost-0.11.9",
+    "rules_rust_prost__prost-types-0.11.9",
+    "rules_rust_prost__protoc-gen-prost-0.2.2",
+    "rules_rust_prost__protoc-gen-tonic-0.2.2",
+    "rules_rust_prost__tokio-1.28.2",
+    "rules_rust_prost__tokio-stream-0.1.14",
+    "rules_rust_prost__tonic-0.9.2",
+    "rules_rust_test_load_arbitrary_tool",
+    "rules_rust_tinyjson",
+    "rules_rust_toolchain_test_target_json",
+    "rules_rust_util_import__aho-corasick-0.7.15",
+    "rules_rust_util_import__lazy_static-1.4.0",
+    "rules_rust_util_import__proc-macro2-1.0.33",
+    "rules_rust_util_import__quickcheck-1.0.3",
+    "rules_rust_util_import__quote-1.0.10",
+    "rules_rust_util_import__syn-1.0.82",
+    "rules_rust_wasm_bindgen__anyhow-1.0.71",
+    "rules_rust_wasm_bindgen__assert_cmd-1.0.8",
+    "rules_rust_wasm_bindgen__diff-0.1.13",
+    "rules_rust_wasm_bindgen__docopt-1.1.1",
+    "rules_rust_wasm_bindgen__env_logger-0.8.4",
+    "rules_rust_wasm_bindgen__log-0.4.19",
+    "rules_rust_wasm_bindgen__predicates-1.0.8",
+    "rules_rust_wasm_bindgen__rayon-1.7.0",
+    "rules_rust_wasm_bindgen__rouille-3.6.2",
+    "rules_rust_wasm_bindgen__serde-1.0.171",
+    "rules_rust_wasm_bindgen__serde_derive-1.0.171",
+    "rules_rust_wasm_bindgen__serde_json-1.0.102",
+    "rules_rust_wasm_bindgen__tempfile-3.6.0",
+    "rules_rust_wasm_bindgen__ureq-2.8.0",
+    "rules_rust_wasm_bindgen__walrus-0.19.0",
+    "rules_rust_wasm_bindgen__wasm-bindgen-0.2.88",
+    "rules_rust_wasm_bindgen__wasm-bindgen-cli-support-0.2.88",
+    "rules_rust_wasm_bindgen__wasm-bindgen-shared-0.2.88",
+    "rules_rust_wasm_bindgen__wasmparser-0.102.0",
+    "rules_rust_wasm_bindgen__wasmprinter-0.2.60",
+    "rules_rust_wasm_bindgen_cli",
+)
+
+rust = use_extension("//rust:extensions.bzl", "rust")
+
+# Allow us to run, for example, "bazel build //tools/runfiles" with bzlmod.
+# Register it as a dev dependency so that we don't force this toolchain on
```

Comment:
This comment is no longer accurate, but that doesn't have to block a merge.

Reply:
@fmeum I created [an issue](https://github.com/bazelbuild/rules_rust/issues/2369) to track the removal of the comment. Unfortunately, I cannot resolve this conversation so that this will merge. Can you resolve it?

Reply:
This would require adding a patch file. I can merge as is, it doesn't really matter.